queue_services/business-pay/src/business_pay/resources/pay_filer.py

```diff
@@ -168,41 +169,42 @@ class PaymentToken:
+    try:
         if filing.effective_date <= filing.payment_completion_date:
-            try:
-                flag_on = flags.is_on("enable-sandbox")
-                logger.debug(f"enable-sandbox flag on: {flag_on}")
-                # use Pub/Sub if FF on, otherwise NATS
-                if flag_on:
-                    data = create_gcp_filing_msg(filing.id)
-
-                    ce = SimpleCloudEvent(
-                        id=str(uuid.uuid4()),
-                        source='business_pay',
-                        subject='filing',
-                        time=datetime.now(timezone.utc),
-                        type='filingMessage',
-                        data = data
-                    )
-                    topic = current_app.config.get('BUSINESS_FILER_TOPIC')
-                    gcp_queue.publish(topic, to_queue_message(ce))
-                    logger.debug(
-                        f"Filer pub/sub message: {str(ce)}"
-                    )
-                else:
-                    filer_topic = current_app.config["FILER_PUBLISH_OPTIONS"]["subject"]
-                    queue_message = create_filing_msg(filing.id)
-                    logger.debug(f"Filer NATS message: {queue_message}")
-                    # await queue.publish(subject=filer_topic, msg=queue_message)
-                    queue.publish_json(subject=filer_topic, payload=queue_message)
-            except Exception as err:
+            flag_on = flags.is_on("enable-sandbox")
+            logger.debug(f"enable-sandbox flag on: {flag_on}")
+            # use Pub/Sub if FF on, otherwise NATS
+            if flag_on:
+                data = create_gcp_filing_msg(filing.id)
+
+                ce = SimpleCloudEvent(
+                    id=str(uuid.uuid4()),
+                    source='business_pay',
+                    subject='filing',
+                    time=datetime.now(timezone.utc),
+                    type='filingMessage',
+                    data = data
+                )
+                topic = current_app.config.get('BUSINESS_FILER_TOPIC')
+                gcp_queue.publish(topic, to_queue_message(ce))
                 logger.debug(
-                    f"Publish to Filer error: {err}, for pay-id: {payment_token.id}")
+                    f"Filer pub/sub message: {str(ce)}"
+                )
+            else:
+                filer_topic = current_app.config["FILER_PUBLISH_OPTIONS"]["subject"]
+                queue_message = create_filing_msg(filing.id)
+                logger.debug(f"Filer NATS message: {queue_message}")
+                # await queue.publish(subject=filer_topic, msg=queue_message)
+                queue.publish_json(subject=filer_topic, payload=queue_message)
+    except Exception as err:
+        logger.debug(
+            f"Publish to Filer error: {err}, for pay-id: {payment_token.id}")
+        # debug
+        logger.debug(traceback.format_exc())
```

Comment:
Temporarily add it to get complete stack trace message, will revert later.

Reply:
I'd keep it in there, stack traces are really handy